### PR TITLE
[Support Requests] Remove dropped ticket sections

### LINKED ISSUE #8334: [Support Requests] Remove My tickets and WCPayment sections
We have decided that we will no longer support the "My Tickets" and "Woo Payment" section. We should delete it.

### PR DESCRIPTION
**Pull request by ThomazFB**
Summary
==========
Fix issue #8334 by removing the `My Tickets` and `Contact Payment Support` sections from the app Help & Support  view. They won't be needed anymore following the new Ticket creation structure we introduced.

Screenshots
==========
| Before  | After |
| ------------- | ------------- |
| Screenshot_20230301_201200 | Screenshot_20230301_200927 |

How to Test
==========
1. Access the `Help & Support` area of the app and verify that both `My Tickets` and `Contact Payment Support` sections are no longer available.
2. Verify that accessing the `Contact Support`section still works as expected.

Update release notes:

- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/src/main/res/layout/activity_help.xml [hunk 1]
```diff
@@ -59,20 +59,6 @@
...
-                android:layout_height="wrap_content"
-                app:optionTitle="@string/support_contact_payments"
-                app:optionValue="@string/support_contact_payments_detail"/>
-
-            <com.woocommerce.android.ui.prefs.WCSettingsOptionValueView
-                android:id="@+id/myTicketsContainer"
-                android:layout_width="match_parent"
-                android:layout_height="wrap_content"
-                app:optionTitle="@string/support_my_tickets"
-                app:optionValue="@string/support_my_tickets_detail"/>
```
Comment (wzieba) [inline WooCommerce/src/main/res/layout/activity_help.xml:74]: It looks like those strings are not used anymore, should we remove them?

Comment (ThomazFB) [reply WooCommerce/src/main/res/layout/activity_help.xml:74]: True! Good catch, thanks!. fixed here: https://github.com/woocommerce/woocommerce-android/pull/8432/commits/71849fc14ae926406de06be8f9b2a5c09c2248ad

**Comment by ThomazFB** [comment]
@wzieba is there something else you think we should change for this PR or it can be approved? Sorry for pushing the review, it's because I'm trying to prepare all PRs before the release cut tomorrow.

**Review by wzieba — APPROVED** [review]
Thank you for the pink @ThomazFB ! Looks good (I should've approved on the first review!)

**Comment by ThomazFB** [comment]
Thank you so much for the review and fast response, @wzieba 🙇🙇🙇

**Pull request merged**